tests/system/conftest.py

```diff
@@ -42,38 +40,6 @@ def _credentials():
     )
 
 
-class TestConf(UnorderedComparator, BackendTest, RoundAwayFromZero):
```

Comment:
These don't appear to be used anywhere. I think it's for running some tests that we aren't running, but I'm not sure. https://github.com/ibis-project/ibis-bigquery/issues/30

Reply:
This is for the tests for all backends in `ibis/backends/tests/`, those should be deployed with Ibis and test external backends also with them.

I'd leave this for now.

Reply:
Should I move it to another location, e.g. `ibis_bigquery/tests` ? It's going to be tough to use this conftest module cross-repo with the way I've been organizing the tests into separate system & unit tests

Reply:
I don't see a problem with it.

Also, I think most of what's defined in this class should be in the `Backend` class, like and attribute to know how rounding is performed for the backend...

So this will probably require changes before those tests can run. Not a big deal to remove if it's causing issues either, but eventually some of this should be needed to run those tests.

Reply:
I'm moving

```
     supports_divide_by_zero = True
    supports_floating_modulus = False
    returned_timestamp_unit = 'us'
```

to the Backend class for now. TBD how to indicate `UnorderedComparator` and `RoundAwayFromZero`.

Possible we'll re-introduce the `TestConf` class at some point, but I think that'd probably belong in `ibis_bigquery.testing` or something and not here. Plus, I really would want there to be backwards compatibility with 1.4.0 (though maybe if `TestConf` lives in a module not strictly required by all tests like conftest.py, it would be fine without a backcompat module)